build/ical.js

```diff
@@ -1384,12 +1384,16 @@ ICAL.stringify = (function() {
       result += stringify.property(props[propIdx], designSet) + LINE_ENDING;
     }
 
-    var comps = component[2];
```

Comment:
I think it would be easier to read to just do `var comps = component[2] || [];`. r+ with that change.